MapboxCoreNavigation/RouteController.swift

```diff
@@ -567,8 +563,12 @@ extension RouteController: CLLocationManagerDelegate {
         if let coordinates = routeProgress.currentLegProgress.currentStep.coordinates {
             let userDistanceToManeuver = Polyline(coordinates).distance(from: location.coordinate)
             
-            guard recentDistancesFromManeuver.count <= 3 else {
-                return false
+            let totalDistances = recentDistancesFromManeuver.reduce(0) { $0 + $1 }
```

Comment:
Nit: this is equivalent to `reduce(0, +)`, because operators like `+` can be used wherever a function can be used.